extensions/wikia/AdEngine3/src/slots.js

```diff
@@ -274,7 +274,7 @@ export default {
 		slotService.setState('hivi_leaderboard', false);
 		slotService.setState('top_leaderboard', true);
 		slotService.setState('top_boxad', isTopBoxadApplicable());
-		slotService.setState('incontent_boxad_1', true);
+		slotService.setState('incontent_boxad_1', isTopBoxadApplicable());
```

Comment:
As now the function refers to two slots maybe we should call it `isRightRailApplicable` or sth like this

Reply:
Done.